src/cli.js

```diff
@@ -13,6 +13,11 @@ program
     'set the port to serve on. defaults to 1234',
     parseInt
   )
+  .option(
+    '-w, --websocket-port <port>',
```

Comment:
can we call the option `--hmr-port` so it's clear what websocket is being referred to?

Reply:
yeah, do you want it to be `-h, --hmr-port`?

Reply:
sounds good.